### PR TITLE
typo: overrided -> override

### PR DESCRIPTION
**Pull request by nschonni**
Could also be "overridden"

### CODE REVIEW

**Comment by PRMerger19** [comment]
@nschonni : Thanks for your contribution! The author, @, has been notified to review your proposed change.

**Comment by TrishaMc** [comment]
Hi @Saisang: 
Please confirm that this PR is ready for merge. 
Thank you.
Trisha

**Comment by Saisang** [comment]
@ghogen could you review it? I'm not a reviewer of this project.

**Comment by gewarren** [comment]
@nschonni I think "overriddenAttributes" would be better here. Can you also please confirm that this still compiles? For non-code comment changes in code snippets, we need to verify that the code still compiles after renaming variables etc.

**Comment by PRMerger9** [comment]
@nschonni : Thanks for your contribution! The author, @, has been notified to review your proposed change.

**Comment by nschonni** [comment]
Rebased for "overridenAttributes", but haven't had a chance to try building it again

**Comment by PRMerger9** [comment]
@nschonni : Thanks for your contribution! The author, @, has been notified to review your proposed change.

File: docs/vs-2015/snippets/visualbasic/VS_Snippets_Wpf/StrokeSnippets/VisualBasic/CustomRenderedStroke.vb [hunk 1]
```diff
@@ -55,13 +55,13 @@ Namespace StrokeSnippets_VB
 
 
         Protected Overrides Sub DrawCore(ByVal context As System.Windows.Media.DrawingContext, _
-                    ByVal overridedAttributes As DrawingAttributes)
-            Dim strokeBrush As New SolidColorBrush(overridedAttributes.Color)
+                    ByVal overridenAttributes As DrawingAttributes)
```
[the remark this answers is not in the document]
Comment (nschonni) [reply docs/vs-2015/snippets/visualbasic/VS_Snippets_Wpf/StrokeSnippets/VisualBasic/CustomRenderedStroke.vb]: lol, fixed

**Comment by nschonni** [comment]
Pulled it down and opened it in 2015. It builds, but apparently is actually a VS2008 solution file :stuck_out_tongue_closed_eyes:

**Comment by gewarren** [comment]
@nschonni Thanks for building it!

**Pull request merged**